src/Import/NoFoundation.hs

```diff
@@ -1,12 +1,9 @@
 module Import.NoFoundation
-    ( module Import.NoFoundation
-    , module Import
+    ( module Import
     ) where
 
 import ClassyPrelude.Yesod   as Import
 import Settings              as Import
 import Yesod.Core.Types      as Import (loggerSet)
 import Yesod.Default.Config2 as Import
 import Control.Category      as Import ((>>>), (<<<))
-
-type One = Succ Zero
```

Comment:
Right, ok. I would prefer to keep exporting `One` from here, I think. If there's no neat way of doing that then I think I'd rather not include this change in this pull request.

It has just also occurred to me that some form of the `maximum` function should be exported from here too, so I'd rather not have the explicit MinLen import in whichever other module it was.